db/system_keyspace.cc

```diff
@@ -2860,10 +2860,13 @@ future<system_keyspace::view_builder_version_t> system_keyspace::get_view_builde
         co_return view_builder_version_t::v1;
     }
     auto& str = *str_opt;
```

Comment:
> Add an intermediate phase to the view builder migration to v2 where we
write to both the old and new table in order to not lose writes during
the migration.

Can we just stop view building while we migrate?

Reply:
I considered it and it didn't seem a better option at that time due to possible impact to availability of views, and error handling seemed a little more complicated if something goes wrong during the migration.